### ISSUE TITLE
selection of a chosen term clears the term

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/solid-crs#479]
If you open the term dialogue to view a previously selected term you loose the selection when you click on the box where term is displayed. This is unwanted behavior because by it does really remove the link when you choose 'bevestigen'. I would have expected opening a new window with the URI of the term.

**Comment by lem-onade** [comment]
Hey @EnnoMeijers!

I've changed this so that the URI displayed under the name is clickable. (See image)
Clicking this link opens it in a new tab and does not deselect the clicked term. Is this what you had in mind?

![image](https://user-images.githubusercontent.com/32491056/135068650-05e4b7ec-7799-4d11-97a5-e6e6f143855f.png)

**Comment by EnnoMeijers** [comment]
Yes, that looks fine to me although we might consider adding a separate deselect option to make this deliberate action? I guess it is up to the tests with real users to find out if this is clear enough now.
